### ISSUE TITLE
Simplified login for single remote auth endpoint

### ISSUE THREAD

**Issue by bricas** [IQSS/dataverse#4869]
The login procedure for remote auth when only one provider is listed seems overly complex:

![screenshot from 2018-07-19 11-41-30](https://user-images.githubusercontent.com/18400/42949815-0806c478-8b49-11e8-8100-df0943cdff52.png)

The select list and "type" options could be hidden, IMHO.

**Comment by pdurbin** [comment]
@bricas you're right, of course. My only hesitation is that I haven't heard a lot of complaints about this and other issues seem more pressing.

Are you still interested in this?

How about others? Still interested?

**Comment by bricas** [comment]
I would say our desire to have that form simplified for single-institution cases is still there. Not urgent, but a nice UX enhancement IMO.

**Comment by cmbz** [comment]
2024/09/16: ABR review, keeping for future.
